yellowbrick/cluster/silhouette.py

```diff
@@ -97,13 +104,15 @@ class SilhouetteVisualizer(ClusteringScoreVisualizer):
     >>> model.poof()
     """
 
-    def __init__(self, model, ax=None, **kwargs):
+    def __init__(self, model, ax=None, colormap='set1', colors=None, **kwargs):
```

Comment:
Matplotlib allows for users to specify parameters such as colormap and colors through kwargs - therefore - the addition you included above will not be required.  We can then use the kwargs below to read in the user defined parameters and use defaults if not specified.

Reply:
based on @rebeccabilbro's suggestion change the above to:

def __init__(self, model, ax=None, colors=None, **kwargs):